minishift/minishift
# Issue #2725 Do not apply templates in xpaas addon one by one

Fixes #2725 


Steps to test the pull request

  1. minishift start
  2. time minishift addons apply xpaas
  3. time should go from current 30s-40s to new time of 4s-9s in case of this PR
